### PR TITLE
object-counter component

### PR DESCRIPTION
**Pull request by RocioDelJ**
new object counter component

### CODE REVIEW

File: src/components/object-counter/object-counter.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  private async countItems(): Promise<void> {
+    const blobImage = this.convertBase64ToBlob(this.imageSrc, "image/jpg");
+    const file = new File([blobImage], 'image', { type: 'image/jpg'});
+    let formData = new FormData();
+    formData.append('image', file);
+    await this.sendCountRequest(formData);
+  }
+
+  sendCountRequest(formData: FormData) : Promise<any> {
```
Comment (marcelpujol) [inline src/components/object-counter/object-counter.tsx:103]: private sendCountRequest

File: src/components/object-counter/object-counter.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        if (request.response) {
+          const response = JSON.parse(request.response);
+          this.counted = response.totalDetected;
+          this.showCountedLabel = true;
+        };
+      }
+    });
+  }
+
+  convertBase64ToBlob(data, type): Blob {
```
Comment (marcelpujol) [inline src/components/object-counter/object-counter.tsx:119]: private convertBase64ToBlob

File: src/components/object-counter/object-counter.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.field-container {
```
Comment (marcelpujol) [inline src/components/object-counter/object-counter.scss:1]: You need to wrap everything to .object-counter-container in order to apply all this scss only in this component. We have same classes such as '.field-container' in other components.

File: package.json [hunk 4]
```diff
@@ -1,9 +1,8 @@
 {
   "name": "@cotecna/stencil-components",
-  "version": "0.0.1-beta.1",
+  "version": "0.0.1-alpha.5",
```
Comment (marcelpujol) [inline package.json:3]: Do we need to publish an alpha version? Or maybe we need to create a stable one, such as: 0.0.1

**Review by Falasvido — DISMISSED** [review]
Only minor things and comments that other reviewers already made. Great job!

File: src/components/object-counter/object-counter.tsx (hunk 1 above)
Comment (RocioDelJ) [reply src/components/object-counter/object-counter.tsx]: done

**Review by javierpardollama — APPROVED** [review]

File: capacitor.config.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import { CapacitorConfig } from '@capacitor/cli';
```
Comment (Xelit3) [inline capacitor.config.ts:1]: Do we need a capacitor configuration file here?

File: package.json [hunk 6]
```diff
@@ -1,9 +1,8 @@
 {
```
Comment (Xelit3) [inline package.json:1]: please undo changes and left this file as it is in main.

After that, knowing that this project still requires some missing points to be considered a release package, I recommend specifying version 0.0.2-beta.1

**Review by Xelit3 — COMMENTED** [review]
Please check configuration files before completing the merge

File: capacitor.config.ts (hunk 5 above)
Comment (RocioDelJ) [reply capacitor.config.ts:1]: yes it is necessary!

File: package.json (hunk 6 above)
Comment (RocioDelJ) [reply package.json:1]: done

**Review by javierpardollama — APPROVED** [review]

**Review by Xelit3 — APPROVED** [review]
Approved! good job @RocioDelJ

**Review by marcelpujol — DISMISSED** [review]

File: stencil.config.ts [hunk 7]
```diff
@@ -35,5 +44,11 @@ export const config: Config = {
       includePaths: ['./node_modules'],
       injectGlobalPaths: ["src/styles.scss"]
     })
-  ]
+  ],
+  //Uncomment to debug
+  // devServer: {
+  //   reloadStrategy: 'pageReload',
+  //   port: 3333,
+  //   address: 'http://192.168.0.11:3333'
```
Comment (javierpardollama) [inline stencil.config.ts]: We should not include hard-coded IPs on public Repositories

**Review by javierpardollama — CHANGES_REQUESTED** [review]

**Review by marcelpujol — DISMISSED** [review]

File: stencil.config.ts (hunk 7 above)
Comment (marcelpujol) [reply stencil.config.ts]: fixed!

**Review by marcelpujol — APPROVED** [review]

**Review by javierpardollama — APPROVED** [review]

**Pull request merged**